zim/plugins/equationeditor.py

```diff
@@ -21,6 +23,8 @@ class InsertEquationPlugin(PluginClass):
 		'name': _('Insert Equation'), # T: plugin name
 		'description': _('''\
 This plugin provides an equation editor for zim based on latex.
+Equations are stored as separate files in the attachments folder,
+or in the wiki page itself.
```

Comment:
nit, I'd prefer a "can" to imply user choice, eg. "can be stored either in-line, or as separate files in the attachments folder"